src/app/code/community/Shopgate/Framework/etc/config.xml

```diff
@@ -226,6 +226,21 @@
                 </observers>
             </adminhtml_init_system_config>
 
+            <controller_front_init_before>
+                <observers>
+                    <init_v2_controller>
+                        <type>model</type>
+                        <class>shopgate/route_observer</class>
```

Comment:
I am thinking of moving this under one folder.
Maybe call the folder `controllers` or `routes` and the class names based on what they rewrite, e.g. V2Observer, CmsMapObserver. I am thinking we may have more in the future. This will also help containing our huge amount of folders :D

Reply:
I would do it the other way round, if we want to change it. Like having an observer folder in models and files like V2 and CmsMap in there, e.g. Shopgate_Framework_Model_Observer_CmsMap . Lets talk about it in the daily.